### PR TITLE
[Schema] Clarify that not required -> optional

### PR DESCRIPTION
**Pull request by RobDolinMS**
Signed-off-by: Rob Dolin <robdolin@microsoft.com>

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -175,6 +175,8 @@ Relative references used in $ref are processed as per [JSON Reference](https://t
 
 ### Schema
 
+In the following description, if an object is not explicitly **Required** or described with a MUST or SHALL, it can be considered OPTIONAL.
```
Comment (ePaul) [inline versions/3.0.md]: object → property/field?

Comment (webron) [inline versions/3.0.md]: Should be "property". @RobDolinMS can you fix?

**Comment by RobDolinMS** [comment]
Thanks @ePaul and @webron.  I've now updated to include:
 * field - used in spec tables
 * property - used in spec paragraphs
 * object - used in spec section headers
If this looks good, please merge (or if more edits are desired, please comment)

**Comment by webron** [comment]
@RobDolinMS I stand corrected it should be `field`, but let's keep it at that. "**Required.**" is pretty much used only in field descriptions.

**Comment by RobDolinMS** [comment]
Thanks @webron.  The PR has been updated to just `field`

**Pull request merged**